incubator/hnc/internal/validators/object_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes/fake"
```

Comment:
You should be able to fake the controller-runtime client fairly easily since it's an interface. The definition is here: https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/client#Client. Obviously the only function you need to implement is `Get`.

Reply:
Thanks @adrianludwin for the review. I will work on these and update the PR.

Reply:
Updated in latest commit